src/oc_time.erl

```diff
@@ -27,9 +27,9 @@
 convert_units({Val, SourceUnit}, TargetUnit) ->
     Val * to_ms(SourceUnit, TargetUnit).
 
-to_ms(_Unit, _Unit) ->
-    1;
 to_ms(min, ms) ->
     60000;
 to_ms(sec, ms) ->
-    1000.
+    1000;
+to_ms(Unit, Unit) ->
```

Comment:
_Unit caused an error with Erlang24x compilation.  Also, I think the underscores are a bug because they are "don't cares" for the inputs and allow any mix of units to return a value of 1 in that clause.  It seems the intent should be to only return a 1 if the units match, e.g. there are 60000 ms in a min, and 1000 ms in a sec, but there is only 1 min in a minute, 1 sec in a sec, and 1 ms in a ms.  So it seems we DO CARE (vs _Dont _Care) what the inputs are to return a value of 1.

I was worried the don't cares would mask a bug.  The enforcing of same unit types to get a return value of 1 will error out on certain mixes, but will (if it ever happens) expose whatever bug.

Or perhaps you and Marc have context I don't have about how and why this was written?  Or you are seeing something I'm missing?